.pre-commit-config.yaml

```diff
@@ -0,0 +1,16 @@
+# Copyright (c) 2023 Sebastian Pipping <sebastian@pipping.org>
+# Licensed under the MIT license
+
+repos:
+  - repo: https://github.com/astral-sh/ruff-pre-commit
+    rev: v0.1.7
+    hooks:
+      - id: ruff
+      - id: ruff-format
```

Comment:
```suggestion
      - id: ruff-format
        args: [ --check, --diff ]
```

Reply:
@jairhenrique I think the current CI output shows that we already get a diff and that hence this is not needed, but I can add them for an experiment to see how the results differ with these, give me a second…

Reply:
@jairhenrique actually I just noticed that this addition will break auto-formatting. Currently you can use pre-commit to have it do the formatting for you, with this change adding `--check` you no longer can. So status quo is actually what we want, regarding this detail. I have add `--diff` as suggested though, since it may be of help during local development. Coming up with next push in a second